norddrop/src/ffi/mod.rs

```diff
@@ -97,23 +110,27 @@ pub extern "C" fn norddrop_new_transfer(
     }
 }
 
-/// @brief Destroy the libdrop instance
+/// Destroy the libdrop instance.
 ///
-/// @param dev   Pointer to the instance
+/// # Arguments
+///
+/// * `dev` - Pointer to the instance.
 #[no_mangle]
+#[allow(clippy::not_unsafe_ptr_arg_deref)]
```

Comment:
@LukasPukenis Are you aware that this is cheating? You should mark the function as `unsafe`

Reply:
You're right. Though I am interested why clippy suddenly out of the blue started to complain about it.